e2e/metadata-panel.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, test } from "./common";
+
+test("it should be possible to open a metadata panel by clicking on elements in the editor", async ({
+  actions,
+  selectors,
+}) => {
+  await actions.chart.createFrom(
+    "https://energy.ld.admin.ch/sfoe/bfe_ogd84_einmalverguetung_fuer_photovoltaikanlagen/6",
+    "Prod"
+  );
+
+  const checkKantonDescription = async () => {
+    const kantonDimensionDescription = await selectors.panels
+      .metadata()
+      .within()
+      .findByText("Kanton, in welchem die geförderten Anlagen stehen");
+
+    expect(kantonDimensionDescription).toBeDefined();
```

Comment:
Nit: I am not sure it is necessary, since the find will throw otherwise. At least it's explicit. Just a thought, not something to fix.